app/controllers/residential_address_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class ResidentialAddressController < StandardStepsController
+  def edit
+    @step = step_class.new(snap_application_attributes.slice(*step_attrs))
+  end
```

Comment:
Correct me if I'm wrong, but isn't this action's contents already in `StandardStepsController`?